Cargo.toml.orig

```diff
@@ -0,0 +1,43 @@
+[package]
```

Comment:
Please remove all extra files.

Reply:
I though it was strange, that when I created this PR, those "orig" files appeared. I'm not entirely why those where generated, but I'll remove them.